argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/opf-vault.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: opf-vault
+spec:
+  project: operate-first
+  source:
+    repoURL: https://github.com/operate-first/apps.git
+    targetRevision: HEAD
+    path: vault/overlays/moc/smaug
+  destination:
+    name: smaug
```

Comment:
Hrmm I was under the impression we would be deploying vault in the infra cluster, as this would be the one instance all clusters use to fetch secrets. No? 

@larsks